sdk/storage/storage-blob/src/generatedModels.ts

```diff
@@ -56,8 +56,8 @@ export {
   PageBlobUploadPagesResponse,
   PageBlobUploadPagesFromURLResponse,
   PageBlobClearPagesResponse,
-  PageBlobGetPageRangesResponse,
-  PageBlobGetPageRangesDiffResponse,
```

Comment:
Why dont we want these to be exported from here?

Reply:
Thanks for pointing this out 😄 

I thought that by adding these here they would get exposed to the API, I just confirmed that it is not the case, so I'm following your suggestion of keeping these here.